cedar-policy-validator/src/schema.rs

```diff
@@ -253,7 +253,7 @@ impl ValidatorSchema {
         fragments: impl IntoIterator<Item = ValidatorSchemaFragment>,
         extensions: Extensions<'_>,
     ) -> Result<ValidatorSchema> {
-        let mut type_defs = HashMap::new();
+        let mut common_types = HashMap::new();
         let mut entity_type_fragments: HashMap<EntityType, _> = HashMap::new();
         let mut action_fragments = HashMap::new();
 
```

Comment:
it won't let me comment on the exact line, by "map of declared typedefs" in the comment a couple lines below

Reply:
Good catch.  I had grepped `TypeDefs` and `type_defs` and `typ_defs` and some other things, but hadn't grepped `typedefs`